src/StickerService.ts

```diff
@@ -50,6 +53,48 @@ function buildWallpaperCss({
     background-size: cover !important;
   }
 
+  /* Output panel */
+  .monaco-workbench .part.panel > .content .monaco-editor .monaco-editor-background,
+  .margin-view-overlays, 
+  .margin, 
```

Comment:
Oh, and these should probably be better constricted to the specific places and not change *all* margins.

Reply:
So which margins where these necessary for then?